bleak/__init__.py

```diff
@@ -154,11 +155,16 @@ def __init__(
             **kwargs,
         )
 
-    async def __aenter__(self):
+    async def __aenter__(self) -> "BleakScanner":
```

Comment:
We should be able to drop the quotes since we have `from __future__ import annotations` in this file already.